.gitignore

```diff
@@ -295,7 +295,6 @@ __pycache__/
 *.xsd.cs
 
 # Unencrypted file
-Snowflake.Data.Tests/parameters.json

```

Comment:
please, don't do this - you can by mistake commit credentials that you wouldn't like to share

Reply:
I think we can commit, it's done the same way for i.e. jdbc and nodejs - it has standard params for local environment

Reply:
@sfc-gh-knozderko @sfc-gh-mhofman what's your opinion here? I just want to avoid situation when by mistake someone push their credentials.

Reply:
pre-commit would be an option or support for env variables in this file